src/data/mystery-encounters/dialogue/getting-lost-at-the-sea-dialogue.ts

```diff
@@ -0,0 +1,50 @@
+import MysteryEncounterDialogue from "#app/data/mystery-encounters/mystery-encounter-dialogue";
+
+const namepsace = "mysteryEncounter:gettingLostAtTheSea";
```

Comment:
```suggestion
const namepsace = "mysteryEncounter:lostAtSea";
```

Following Imperial's suggestion in the Discord

Reply:
I think I'll let you refactor though lol

Reply:
I’ll get to it at some Point. Originally I had that name but I just took the literal name then 😁

Reply:
@AsdarDevelops done